dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.mpReactiveMessaging-mpConfig2.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpReactiveMessaging-mpConfig2
+visibility=private
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.mpReactiveMessaging-1.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.mpConfig-2.0) \
+                                                          ))"
+-bundles=com.ibm.ws.io.smallrye.reactive.messaging-provider.config20
+IBM-Install-Policy: when-satisfied
+kind=ga
```

Comment:
If this wasn't a draft, I would say set this to beta

Reply:
Yeah, it should be.

Reply:
Unfortunately, I don't think we can do that. mpRM-1.0 and mpConfig-2.0 are already available. If we make the changes to mpRM-1.0 to tolerate mpConfig-2.0, then we need this feature to be present, otherwise you have a server which starts but then the bundles won't resolve.

Reply:
I think the only way we could beta this feature would be to have a separate feature which is identical to mpRM-1.0 except that it also tolerates mpConfig-2.0. To try out the functionality, users would have to enable the separate feature in place of mpRM-1.0.

Reply:
mpConfig-2.0 is still beta

Reply:
ah, that's true, it should match mpConfig-2.0

Reply:
well, it might go to GA straightaway though, 21.0.0.3.